Lib/guile/guile_scm_run.swg

```diff
@@ -176,6 +176,9 @@ SWIG_Guile_ConvertPtr(SCM s, void **result, swig_type_info *type, int flags)
   if (SCM_NULLP(smob)) {
     *result = NULL;
     return SWIG_OK;
+  } if (SCM_POINTER_P(s)) {
```

Comment:
I'd suggest an `else` before the `if` here.

While it doesn't actually make any difference as the code stands (because there preceding block ends with a `return`), having `} if` seems a trap for future developers if that `return` is removed, and `else if` would be consistent with the code below what you've inserted.